de.j0chn.nextcloud_password_client.json

```diff
@@ -0,0 +1,107 @@
+                }
+            ]
+        },
+        {
+            "name": "libsecret",
+            "buildsystem": "meson",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=release",
+                "-DBUILD_SHARED_LIBS=ON"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/GNOME/libsecret/archive/refs/tags/0.20.4.tar.gz",
+                    "sha256": "ca34e69b210df221ae5da6692c2cb15ef169bb4daf42e204442f24fdb0520d4b"
+                }
+            ]
+        },
+        {
+            "name": "nextcloud_password_client",
+            "buildsystem": "simple",
+            "only-arches": [
+                "x86_64"
+            ],
+            "build-commands": [
+                "ls nextcloud_password_client",
+                "cp -r nextcloud_password_client /app/nextcloud_password_client",
+                "chmod +x /app/nextcloud_password_client/",
+                "install -Dm644 npc_logo.png /app/share/icons/hicolor/scalable/apps/de.j0chn.nextcloud_password_client.nextcloud_password_client.svg",
+                "mkdir -p //app/bin",
+                "ln -s /app/nextcloud_password_client/nextcloud_password_client /app/bin/nextcloud_password_client",
+                "install -Dm644 de.j0chn.nextcloud_password_client.desktop /app/share/applications/de.j0chn.nextcloud_password_client.desktop",
+                "install -Dm644 de.j0chn.nextcloud_password_client.metainfo.xml /app/share/appdata/de.j0chn.nextcloud_password_client.metainfo.xml"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "only-arches": [
+                        "x86_64"
+                    ],
+                    "url": "https://gitlab.com/api/v4/projects/30298940/packages/generic/nextcloud_password_client/0.1.0/nextcloud_password_client-linux-x86.tar.gz",
+                    "dest": "nextcloud_password_client",
+                    "sha256": "4aedf8dfe68e57305c9b5f71723b714345982c57f615f4dad6dbfd7eefc7c2d1",
+                    "x-checker-data": {
+                        "type": "json",
+                        "url": "https://gitlab.com/j0chn/nextcloud_password_client/-/releases/",
+                        "version-query": ".[0] | .tag_name | sub(\"v\";\"\")",
+                        "url-query": ".[0] | .assets  | .links[] | select(.name==\"nextcloud_password_client-linux-x86.tar.gz\") | .url"
+                    }
+                },
+                {
+                    "type": "file",
+                    "path": "de.j0chn.nextcloud_password_client.metainfo.xml"
+                },
+                {
+                    "type": "file",
+                    "path": "de.j0chn.nextcloud_password_client.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "npc_logo.png"
```

Comment:
shouldn't that be usptream?

Reply:
Just to be clear, you mean I should take the Logo Form my gitlab repo and not Form the flathub Branche, right?

Reply:
yes, or from the tarball. The distribution has an application icon to go with the .desktop, doesn't it?